dev/pytest/pytest_vdb.sh

```diff
@@ -8,4 +8,5 @@ pytest api/tests/integration_tests/vdb/chroma \
   api/tests/integration_tests/vdb/pgvector \
   api/tests/integration_tests/vdb/qdrant \
   api/tests/integration_tests/vdb/weaviate \
+  api/tests/integration_tests/vdb/es \
```

Comment:
```suggestion
  api/tests/integration_tests/vdb/elasticsearch \
```

Use the full name for path is more clear and extensible .